howso/client/tests/test_client.py

```diff
@@ -211,8 +205,10 @@ def trainee(self, trainee_builder):
                     'datetime': {'type': 'continuous',
                                  'date_time_format': '%Y-%m-%dT%H:%M:%S.%f'}
                     }
-        trainee = Trainee(features=features)
-        trainee_builder.create(trainee, overwrite_trainee=True)
+        trainee = dict(
```

Comment:
Lets just pass these as parameters to the create() call

Reply:
Applies to other instances like this in this file.

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60
and c5525aaf34989005ca900811b14f659ea72fee8e